lib/mongoose_push_web/swagger_plug.ex

```diff
@@ -0,0 +1,23 @@
+defmodule MongoosePushWeb.SwaggerPlug do
+  use Plug.Builder
+
+  @behaviour Plug
+
+  @impl Plug
+  def init(opts) do
+    if(Application.get_env(:mongoose_push, :enable_swagger) == true) do
+      OpenApiSpex.Plug.PutApiSpec.init(opts)
+    end
+  end
+
+  @impl Plug
+  def call(conn, nil) do
```

Comment:
What if `OpenApiSpex.Plug.PutApiSpec.init/1` returns nil? We can't really know for sure, as this is not really documented, and thus may change. We should return from our `init/1` something very specific, maybe very specific atom, and match on that instead.

Reply:
Currently `init` returns `nil` when the flag is set to `false`, so we're not creating all the stuff when we don't expect it to happen.

Reply:
Yeah, but what if `OpenApiSpex.Plug.PutApiSpec.init/1` starts returning `nil`?

Reply:
Then currently this `nil` is propagated like in the previous case. Will change it.